openshift_pipelines/op-release-notes.adoc

```diff
@@ -0,0 +1,36 @@
+//OpenShift Pipelines Release Notes
+include::modules/pipelines-document-attributes.adoc[]
+[id="op-release-notes"]
+= {pipelines-title} Release Notes
```

Comment:
I know you said you were changing the topic map in another PR, but you should still change this title in this PR to be sentence case ("release notes")